docs/pages/User/Configure/model.md

```diff
@@ -0,0 +1,32 @@
+---
+title: The Liqo Tenant Model 
+weight: 1
+---
+
+The peering process in Liqo introduces the possibility to share resources among cluster. Mapping Kubernetes
+resources across different clusters, Liqo proposes a use model to share Kubernetes objects (e.g. primarily pods).
+
+![](/images/home/architecture.png)
+
+## Liqo Tenant Model
+
+* Pods are replicated, not deployments
+* Several objects are kept synchronized between namespaces:
+   * Services/Endpoints
+   * Configmaps/Secrets
+
+## Scheduling Pods on VirtualNodes
+
+To schedule pods on other clusters, you have several options:
+
+* spawn it in a namespace can be extended on another cluster if it is labelled ```liqo.io/enabled=true```
```

Comment:
Not clear. Do you mean:
* spawn it in a namespace that can be alive also in a foreign cluster (this requires the namespace to be labelled ```liqo.io/enabled=true```)